setup/security/authorization/pipeline-permissions/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: single
+title:  "Pipeline Permissions"
+sidebar:
+  nav: setup
+---
+
+{% include alpha version="1.9" %}
+
+{% include toc %}
+
+Pipeline permissions enable automatically triggered pipelines to modify
+resources in protected accounts and applications. They are an alternative
+to manually managing (Fiat Service Accounts)[../service-accounts/].
+
+Without pipeline permissions, a Spinnaker operator first has to first create a
```

Comment:
nit: remove one of the firsts in this sentence.

Reply:
done